src/Umbraco.Infrastructure/HostedServices/ContentVersionCleanup.cs

```diff
@@ -32,7 +32,7 @@ public class ContentVersionCleanup : RecurringHostedServiceBase
             IContentVersionService service,
             IMainDom mainDom,
             IServerRoleAccessor serverRoleAccessor)
-            : base(TimeSpan.FromHours(1), TimeSpan.FromMinutes(3))
+            : base(logger, TimeSpan.FromHours(1), TimeSpan.FromMinutes(1))
```

Comment:
@vsilvar Is the Timespan.FromMinutes change here intentional ? 🤔

Reply:
@Zeegaan Oops, completely unintentional. 😅